test/fixtures/ejson-cloud/web.yaml

```diff
@@ -1,10 +1,17 @@
 ---
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: web
+  labels:
```

Comment:
Is it because `labels` and `selector` become mandatory fields in `apps/v1`?

Reply:
Yeah, this is why I had to add it to a bunch of templates.